src/pages/example/example.tsx

```
@@ -1,41 +1,17 @@
 import React, { memo } from "react";
 
-import { getItems } from "@/entities/doc/api/docApi";
-import type { Part } from "@/entities/doc/model/types";
-import Box from "@/shared/ui/Box";
-import Spinner from "@/shared/ui/Spinner";
+import { Select } from "@/shared/ui";
 
 interface Props {}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)